rocAL/rocAL/source/node_ssd_random_crop.cpp

```diff
@@ -87,10 +94,12 @@ void SSDRandomCropNode::update_node()
         memcpy(labels_buf.data(), _meta_data_info->get_bb_labels_batch()[i].data(), sizeof(int) * bb_count);
         std::vector<float> coords_buf(bb_count * 4);
         memcpy(coords_buf.data(), _meta_data_info->get_bb_cords_batch()[i].data(), _meta_data_info->get_bb_cords_batch()[i].size() * sizeof(BoundingBoxCord));
-        crop_box.h = _crop_height_val[i];
-        crop_box.w = _crop_width_val[i];
-        crop_box.x = _x1_val[i];
-        crop_box.y = _y1_val[i];
+
+        crop_box.b = _y1_val[i] + _crop_height_val[i];
```

Comment:
aren't the coordinates already in ltrb?

Reply:
Only the bounding box co-ordinates are in ltrb format, the image crop co-ordinates would be still in xywh format

Reply:
Yes. I don't see the point of updating crop_box before the while loop since it will be set within the loop.